lib/MusicBrainz/Server/Email.pm

```diff
@@ -50,14 +50,9 @@ sub _user_address
     my ($user, $hidden) = @_;
 
     my $quoted_name = _encode_header($user->name);
+    my $email = $hidden ? 'noreply@musicbrainz.org' : $user->email;
```

Comment:
Instead of using hardcoded noreply address it'd be more consistent to modify/re-use https://github.com/metabrainz/musicbrainz-server/blob/f48a16b81f8ffecdef3552f41b48092c4adc3199/lib/MusicBrainz/Server/Constants.pm#L293
 (it'd need some changes though as format is different)

Reply:
@zas: Moved around in https://github.com/metabrainz/musicbrainz-server/pull/658/commits/119f904c20d7dbb73755144772e25cb22700936c. Technically it is still _hardcoded_, but at least these related constants are defined in a single place.